### PR TITLE
fix: display MyC Interested in selling tooltip only for P1 artists

### PR DESCRIPTION
**Pull request by olerichter00**
[ ](https://www.notion.so/artsy/Adding-a-P1-artwork-tooltip-not-showing-up-5fcae97fdde24d65a2bb25e25eb9c056?pvs=4)

### Description

We want to display "MyC Interested in selling" tooltip only for P1 artists. `artwork.artist?.targetSupply?.priority` is not a boolean and this PR adjusts the check

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Fix display "MyC Interested in selling" tooltip only for P1 artists - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Fix display "MyC Interested in selling" tooltip only for P1 artists - ole)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 3179f4102c137d119caeb6ee0adb26253508a328
</p>

File: src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx [hunk 1]
```diff
@@ -330,7 +330,7 @@ const InfiniteScrollArtworksGrid: React.FC<Props & PrivateProps> = ({
           isMyCollection &&
           !isDismissed(PROGRESSIVE_ONBOARDING_MY_COLLECTION_SELL_THIS_WORK).status &&
           itemIndex === 0 &&
-          !!artwork.artist?.targetSupply?.priority
+          artwork.artist?.targetSupply?.priority === "TRUE"
```
Comment (gkartalis) [inline src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx:333]: 😳 sad that this is not a boolean

**Review by gkartalis — APPROVED** [review]

File: src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx:333]: do we intentionally want to test for `priority` here instead of `isP1`

Comment (MounirDhahri) [reply src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx:333]: just read the interface in MP again - should be fine this way

**Review by nickskalkin — APPROVED** [review]

File: src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx (hunk 1 above)
Comment (olerichter00) [reply src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx:333]: `isP1` is deprecated in favor of this new `priority` enum (https://github.com/artsy/metaphysics/pull/5266/files#diff-3a304b8e0fa043b030a2e5b3040547c6e4008ebebed2a9207ba67d27dc18192dR2025).

Comment (olerichter00) [reply src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx:333]: It's a fancy boolean now, ready for a world beyond binary logic 🦸‍♀️

**Pull request merged**